pkgs/development/ocaml-modules/bitstring/ppx.nix

```diff
@@ -9,7 +9,9 @@ else
 
 buildDunePackage rec {
   pname = "ppx_bitstring";
-  inherit (bitstring) version useDune2 src;
+  inherit (bitstring) version src;
+
+  duneVersion = "3";
```

Comment:
```suggestion
  inherit (bitstring) version duneVersion src;
```

Same for many other package ? It could then be automated (just one sed)

Reply:
What do you mean?

Reply:
I'm afraid that won't work.

Reply:
Because of this:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/ocaml/dune.nix#L42

Feel free to try it and see for yourself.